apps/client/src/components/add-toast/add-toasts.tsx

```diff
@@ -36,17 +59,157 @@ export const AddToast: React.FC<Props> = ({
+    '&:hover': {
+      cursor: 'pointer',
+      transform: 'scale(1.2)',
+    },
+  };
 
   const handleClickAway = () => {
     setIsAddToastDialogOpe(false);
   };
 
+  const [solidsPick, setSolidsPick] = useState<string[]>([]);
+
+  const handleChange = (event: SelectChangeEvent<typeof solidsPick>) => {
+    const {
+      target: { value },
+    } = event;
+    setSolidsPick(typeof value === 'string' ? value.split(',') : value);
+  };
+
+  const solids = [
+    'פלאפל',
+    'פיצות',
+    '',
+    'גלידה',
+    'קרקרים ומטבלים',
+    'פירות',
+    'עוגיות',
+    'עוגות',
+  ];
+
+  const fluids = ['אייס קפה', 'שוקו', '', 'מיץ', 'סודה', 'מים בטעמים', 'קולה'];
+
   return (
     <ThemeProvider theme={darkTheme}>
-      <ClickAwayListener onClickAway={handleClickAway}>
-        <Dialog open={isAddToastDialogOpen} sx={dialogStyle}></Dialog>
-      </ClickAwayListener>
-      ;
+      <Dialog
+        open={isAddToastDialogOpen}
+        sx={dialogStyle}
+        onClose={handleOnClose}
+      >
+        <DialogTitle
+          sx={{
+            padding: '2rem',
+          }}
+          component="h1"
+          align="center"
+        >
+          הוסיפו שתייה לקראת האירוע הבא שלכם!
+        </DialogTitle>
+
+        <DialogContent sx={dialogContentStyle}>
+          <Stack
+            component="form"
+            sx={{
+              gap: '2rem',
+              padding: '1rem',
+              width: '80%',
+            }}
+          >
+            <TextField
+              placeholder="אנא ציין את סיבת השתייה"
+              name="description"
+              required
+            />
+            <FormControl sx={{ width: '100%' }}>
+              <CacheProvider value={cacheRtl}>
+                <InputLabel id="solids-label">בחר את השתייה</InputLabel>
+                <Select
+                  labelId="demo-multiple-name-label"
+                  id="demo-multiple-name"
+                  multiple
+                  value={solidsPick}
+                  onChange={handleChange}
+                  input={<OutlinedInput label="Name" />}
+                >
+                  {fluids.map((fluid) => (
+                    <MenuItem key={fluid} value={fluid}>
+                      {fluid}
+                    </MenuItem>
+                  ))}
+                </Select>
+              </CacheProvider>
+            </FormControl>
+            <FormControl sx={{ width: '100%', direction: 'rtl' }}>
+              <CacheProvider value={cacheRtl}>
+                <InputLabel id="solids-label">בחר את האוכל</InputLabel>
+                <Select
+                  labelId="demo-multiple-name-label"
+                  id="demo-multiple-name"
+                  multiple
+                  value={solidsPick}
+                  onChange={handleChange}
+                  input={<OutlinedInput label="Name" />}
+                >
+                  {solids.map((solid) => (
+                    <MenuItem key={solid} value={solid}>
```

Comment:
Are you sure the key needs to be `solid`.
What if you have the same solid twice? 
I think theres a better way..